src/python/botan.py

```diff
@@ -15,7 +16,7 @@
 """
 Module initialization
 """
-botan = CDLL('libbotan-1.11.so')
+botan = CDLL('libbotan-1.11.dylib')
 
```

Comment:
Oops! My fault - it is not platform-independent. Thanks for catching it. Will fix and re-post.

Regarding "random links" - I thought that a Pull Request with GitHub comments is clear enough? Or should I duplicate on the mailing list what I already put in the GitHub comments? No big deal for me (cut-n-paste works), just want to know what your preference is.

Sent from my iPad

On Aug 12, 2015, at 02:38, Simon Warta <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/python/botan.py<https://github.com/randombit/botan/pull/246#discussion_r36830343>:

> @@ -15,7 +16,7 @@
>  """
>  Module initialization
>  """
> -botan = CDLL('libbotan-1.11.so')
> +botan = CDLL('libbotan-1.11.dylib')
>


I think this should be somehow platform independent. Does that work in Windows as well?

—
Reply to this email directly or view it on GitHub<https://github.com/randombit/botan/pull/246/files#r36830343>.